src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -63,7 +63,6 @@ public IVoidConfiguration WhenArgumentsMatch(Func<ArgumentCollection, bool> argu
 
         public virtual IAfterCallSpecifiedConfiguration DoesNothing()
         {
-            this.RuleBeingBuilt.Applicator = x => { };

```

Comment:
It seems so simple, yet it worries me at the same time for some reason I can't quite put my finger on.

Is there any undesired side effect here? I won't pretend to know what `this.RuleBeingBuilt.Applicator` is or does, but is there any scenario where this will change behaviour and break some existing tests? I.e. is it currently possible to chain `DoesNothing` onto some other configuration calls, which effectively clears some previous config? (Please tell me if I'm talking complete nonsense!)